lib/generators/maintenance_tasks/task_generator.rb

```diff
@@ -9,23 +9,29 @@ class TaskGenerator < Rails::Generators::NamedBase
     desc 'This generator creates a task file at app/tasks and a corresponding '\
       'test.'
 
-    class_option :csv, type: :boolean, default: false,
-      desc: 'Generate a CSV Task.'
+    TASK_TYPES = ['collection', 'csv'].freeze
+    private_constant :TASK_TYPES
```

Comment:
Any reason this needs to be private in addition to the class being private?

Reply:
Habit, since it's really an implementation detail. I can remove it if it's inappropriate.